### DISCUSSION TITLE
Can't create new catch diff when taiko one is opened in editor

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#22140] [Q&A] [answered]
This:

https://user-images.githubusercontent.com/53872073/211896565-6e0b45d0-dc01-47a6-beb8-12a9e100e751.mp4

Works with any beatmap.

[runtime.log](https://github.com/ppy/osu/files/10395313/runtime.log)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/22145
